libs/ledger-live-common/src/families/cosmos/js-synchronisation.ts

```diff
@@ -37,9 +47,12 @@ const txToOps = (info: AccountShapeInfo, accountId: string, txs: CosmosTx[]): Co
   for (const tx of txs) {
     let fees = new BigNumber(0);
 
-    tx.tx.auth_info.fee.amount.forEach(elem => {
-      if (elem.denom === unitCode) fees = fees.plus(elem.amount);
-    });
+    // LIVE-9520
```

Comment:
Is it a jira ticket ? I don't think it's a relevant comment to put in the code

Reply:
See description, I am not sure it is the fix as it can't be reproduced, but it's the only way I can think of we can get this error on init not related to validators

Reply:
Yes got it but putting a jira ticket as a comment is not gonna help. Plus we're technically an open source project jira tickets are not available to all